package.json

```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
     "eslint": "^7.26.0",
-    "grunt": "^1.4.0",
```

Comment:
Can you revert your changes to the `package.json` and `package-lock.json` files? If you look into "reverting commits with git", hopefully it shouldn't be too difficult to undo that commit. If you're running into issues, you can just create a new branch off of develop and try again.

We want to keep PRs to changing one thing at a time. So we just want to make the change outlined in the issue. Upgrading packages is something that we want to do separately so that I'm not testing both the package upgrades _and_ the changes that you needed to make here.